src/test/java/seedu/address/logic/commands/event/ESortCommandTest.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands.event;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.general.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.model.event.DateAndTime.DATE_TIME_FORMATTER;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.common.Name;
+import seedu.address.model.event.EndDateTime;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.StartDateTime;
+
+class ESortCommandTest {
+
+    @Test
+    public void execute_emptyList() {
+        Model model = new ModelManager();
+        ESortCommand command = new ESortCommand();
+        assertCommandSuccess(command, model, ESortCommand.MESSAGE_SUCCESS, model);
+    }
+
+    @Test
+    public void execute_nonEmptyList() {
```

Comment:
If possible, try to reduce the length of this method. Seems abit long. Maybe you can use a for loop/array since there are some similarities for some of the LOC such as `model.addEvent`